k8s/scripts/cis/README.md

```diff
@@ -37,7 +37,7 @@ pip install -r requirements.txt
 Run the script
 
 ```sh
-python3 cis-yaml-to-md.py --input-dir=./kube-bench/cfg/cis-1.24-ck8s --output-dir=../../../docs/src/_parts/cis/
+python3 cis-yaml-to-md.py --input-dir=./kube-bench/cfg/ck8s-cis-1.24 --output-dir=../../../docs/src/_parts/cis/
```

Comment:
you also need to change the branch you pull earlier in this readme to -b ck8s-dqlite to access this new config

Reply:
You are right. I updated the pull branch to be `ck8s-dqlite`.